external/src/test/groovy/org/hawkular/alerts/rest/TriggersITest.groovy

```diff
@@ -521,6 +521,38 @@ class TriggersITest extends AbstractQuarkusITestBase {
         assertEquals(200, resp.status)
     }
 
+    @Test
+    void createFullTriggerWithNullId() {
+        String jsonTrigger = "{\n" +
+                "      \"trigger\":{\n" +
+                "        \"id\": null,\n" +
```

Comment:
I would prefer this "id" line to be gone actually. Just to ensure this doesn't accidentally turn into "null" string.

Reply:
Sure thing

Reply:
done